test/PushNotificationTests/BaseTestSuite.cpp

```diff
@@ -113,6 +114,27 @@ void BaseTestSuite::ChannelRequestUsingRemoteId()
     }
 }
 
+void BaseTestSuite::ChannelRequestCheckExpirationTime()
+{
+    if (PushNotificationManager::Default().IsSupported())
+    {
+        auto channelOperation{ PushNotificationManager::Default().CreateChannelAsync(c_azureRemoteId) };
+        VERIFY_SUCCEEDED(ChannelRequestHelper(channelOperation));
+
+        auto channel{ channelOperation.GetResults().Channel() };
+        auto expirationTime{ channel.ExpirationTime() };
+        auto expiryBound{ winrt::clock::now() + std::chrono::days(31) };
+
+        // Need to add 30 days to match expiration time.
+        VERIFY_IS_LESS_THAN(expirationTime, expiryBound);
```

Comment:
Are both of these in UTC? We need to make an apples to apples comparison.

Reply:
Yes. These are both UTC.